app/javascript/gobierto_dashboards/webapp/containers/shared/Aside.vue

```diff
@@ -1,14 +1,169 @@
 <template>
   <div class="pure-u-1 pure-u-lg-1-4">
     <aside class="dashboards-home-aside--gap">
-      wip filtros
+      <div
+        v-for="filter in filters"
+        :key="filter.id"
+        class="dashboards-home-aside--block"
+      >
+        <Dropdown @is-content-visible="toggle(filter)">
+          <template v-slot:trigger>
+            <BlockHeader
+              :title="filter.title"
+              class="dashboards-home-aside--block-header"
+              see-link
+              @select-all="e => handleIsEverythingChecked({ ...e, filter })"
+              @toggle="toggle(filter)"
+            />
+          </template>
+          <div>
+            <Checkbox
+              v-for="option in filter.options"
+              :id="option.id"
+              :key="option.id"
+              :title="option.title"
+              :checked="option.isOptionChecked"
+              :counter="option.counter"
+              class="dashboards-home-aside--checkbox"
+              @checkbox-change="e => handleCheckboxStatus({ ...e, filter })"
+            />
+          </div>
+        </Dropdown>
+      </div>
     </aside>
   </div>
 </template>
 
 <script>
+import { BlockHeader, Checkbox, Dropdown } from "lib/vue-components";
+import { EventBus } from "../../mixins/event_bus";
+import { filtersConfig } from "../../lib/config.js";
+
 export default {
   name: 'Aside',
-  data() { return {} }
+  components: {
+    Dropdown,
+    BlockHeader,
+    Checkbox,
+  },
+  data() {
+    return {
+      filters: filtersConfig
+    }
+  },
+  props: {
+    contractsData: {
+      type: Array,
+      default: {}
+    }
+  },
+  created(){
+    this.initFilterOptions();
+    this.updateCounters(true);
+
+    EventBus.$on('dc_filter_selected', ({title, id}) => {
+      const filter = this.filters.find(filter => filter.id === id)
+
+      filter.options.forEach(option => {
+        if (option.title === title) {
+          option.isOptionChecked = !option.isOptionChecked;
+        }
+      })
```

Comment:
High error possibility

Reply:
What do you suggest?

Reply:
```suggestion
      const { options = [] } = this.filters.find(( { id: i } ) => id === i) || {};

      options.forEach(option => {
        if (option.title === title) {
          option.isOptionChecked = !option.isOptionChecked;
        }
      })
```

Reply:
👏 thanks, it's much better and it works great.